sql/user-account-management.md

```diff
@@ -0,0 +1,94 @@
+---
+title: TiDB 用户账户管理
+category: user guide
+---
+
+# TiDB 用户账户管理
+
+## 用户名和密码
+
+TiDB 将用户账户存储在 `mysql.user` 系统表里面。每个账户由用户名和 host 作为标识。每个账户可以设置一个密码。
+
+通过 MySQL 客户端连接到 TiDB 服务器，通过指定的账户和密码登陆：
+
+```
+shell> mysql --port 4000 --user xxx --password yyy
```

Comment:
better receive password from user input.
this will put your plain password in shell history, which is dangerous.

btw, seems that ``--password`` requires ``--password=xxx`` instead of space seperated.